app/controllers/items_controller.rb

```diff
@@ -1,28 +1,58 @@
 class ItemsController < ApplicationController
-    def index
-        @items=Item.all
-    end
-    
-    def show
-        @item=Item.find(params[:id])
-    end
-    
-    def new
-        @item=Item.new
-        @facilities=Facility.all
-    end
-    
-    def create
-        @item=Item.new
-        @item.name=params[:item][:name].upcase
-        @item.date=params[:item][:date]
-        @item.description=params[:item][:description]
-        @item.price=params[:item][:price]
-        @item.status=params[:item][:status]
-        @item.facility_id=params[:facility_id]
-        @item.status="pending"
-        @item.save
-        redirect_to items_path
-    end
+  before_action :authenticate_user!, only: [:edit, :update, :destroy, :base_params, :create_params, :update_params]
+  
+  def index
+    @items=Item.where("status = 'approved'")
+  end
+    
+  def show
+    @item=Item.find(params[:id])
+  end
+    
+  def new
+    @item=Item.new
+  end
+    
+  def create
+    @item = Item.create(create_params)
+    @item.status = "pending"
+    @item.save
+    redirect_to "/"
+  end
+    
+  #edit exisiting database entry -- only available to admin
+  def edit
+    @item = Item.find(params[:id])
+  end
+    
+  #update exisiting database entry -- only available to admin
+  def update
+    @item = Item.find(params[:id])
+    @item.update(update_params)
+    @item.status = 'pending'
+    @item.save
+    redirect_to "/admins"
+  end
+    
+  #destroy database entry 
+  def destroy
+    item = Item.find(params[:id])
+    item.destroy
```

Comment:
These should probably be instance variables.

Reply:
We used instance variables in create because someone in slack noted .create only works on instance variables.  However I couldn't find anything related to .destroy (and the feature works as is).
I made instance variables for items and procedures, in their controllers, and it seems to work fine too.  Let me know if you think we should undo this change.

Reply:
I also made instance variables for the destroy method in the facilities controller.